### ISSUE TITLE
Sample Request: Interactive Mind Map web part

### ISSUE THREAD

**Issue by hugoabernier** [pnp/sp-dev-fx-webparts#1191]
## Category

- [ ] Question
- [ ] Bug
- [ ] Enhancement
- [x] Sample request

## Expected or Desired Behavior

I would love to build a web part that makes it easy to display and collaborate on mind maps.

Ultimately, I'd love to make it interactive (even Fluid-like) and make it available to use within SharePoint or teams.

If anybody has any tips or wants to help me build this, I could use the help. 

I have found no suitable libraries to render beautiful mind maps, and I'm not quite ready to create my own mind map editing library yet.

**Comment by Harshagracy** [comment]
Hi @hugoabernier did you check this one - https://www.npmjs.com/package/mind-map

**Comment by hugoabernier** [comment]
Hi @Harshagracy  I did a couple years ago when I first started looking into this.

I think the original issue I had with it was that it broke once inside of a web part... but I've learned a lot since about solving such issues.

I'll give it a try. If it works, is this something you'd be interested in collaborating with me? I think this would be a web part that people would probably use in every day team collaboration -- something that might be good while we're all stranded at home.
